openwrt/nodogsplash/files/etc/config/nodogsplash

```diff
@@ -33,6 +33,15 @@ config nodogsplash
 	#
 	option login_option_enabled '0'
 
+	# Use outdated libmicrohttpd (MHD)
+	# Older versions of MHD convert & and + characters to spaces when present in form data
+	# This can make a PreAuth or BinAuth impossible to use for a client if form data contains either of these characters
+	# eg. in username or password
+	# MHD versions earlier than 0.9.69 are detected.
+	# If this option is set to 0 (default), NDS will terminate if MHD is earlier than 0.9.69
+	# If this option is set to 1, NDS will start but log an error.
```

Comment:
A warning instead of an error sounds better.

Reply:
Yes it does, but I wanted it to show in the log even if debuglevel is set to 0 (silent).
Updating on OpenWrt preserves the config, so if debuglevel _**is**_ set to 0 it will not be seen.

Reply:
ok